nservicebus/sqlserver/addressing_content_sqlserver_[3,3.1).partial.md

```diff
@@ -0,0 +1,39 @@
+### Format
+
+The SQL Server address has following canonical form
+
+```
+table@[schema]
+```
+
+where:
+ * `table` is an unquoted delimited identifier without the surrounding square brackets. Whitespace and special characters are allowed and are not escaped e.g. `my table` and `my]table` are legal values. The identifier is quoted automatically by SQL Server transport when executing the T-SQL statements. `@` is a separator between the table and schema parts and thus is not a valid character. 
+ * `schema` is either an unquoted delimited identifier without the surrounding square brackets or a standard bracket-delimited identifier. In the second form it is always surrounded by brackets and any right brackets (`]`) inside are escaped e.g. `[my]]schema]`. `@` is only allowed in the bracket-delimited form, otherwise it is treated as separator.
+
+
+### Resolution
+
+The address is resolved into a qualified table name that includes both table name and its schema. In the address the table name is the only mandatory part. An address containing only a table name is a valid address e.g. `MyTable`. 
+
+
+#### Schema
+
+Schema name is optional. Even if it is present in the address, it might be overridden by configuration. The algorithm for calculating the schema is following:
```

Comment:
It might be useful when processing old messages (e.g. from SC retries) after the original endpoint has been migrated to a different schema.